animation/effects/MapEffect.java

```diff
@@ -0,0 +1,72 @@
+package animation.effects;
+
+import animation.Animation;
+import screens.Drawable;
+
+import java.awt.*;
+
+public abstract class MapEffect implements Drawable {
```

Comment:
should probably rename from MapEffect to something like Effect, since one of the enum types in this class also has the same name